### ISSUE TITLE
On introduction page, it’s hard to go to previous slide with a mouse/on touch

### ISSUE THREAD

**Issue by ariasuni** [mastodon/mastodon#13700]
### Expected behaviour

There should be a button to easily go back to previous slide. I could be a relatively discreet secondary button.

### Actual behaviour

The only way to go back with a mouse is to click/touch the small dots at the bottom of the screen. On a desktop with a mouse, there’s no way to know that arrows are working too. On mobile, there are not dots a user may not expect sliding to work.

![bildo](https://user-images.githubusercontent.com/2446451/81516854-a9d5c700-9339-11ea-886a-d1d8757cd567.png)

### Steps to reproduce the problem

1. Create a new account
2. Login and see the introduction page
3. Go to the next slide by clicking «Next»
4. There’s no obvious big target to go back

### Specifications

Mastodon v3.1.3

**Comment by mjankowski** [comment]
Closing on assumption that this is resolved in newer versions. Can you re-open with current screenshots if still problem in 4.3?
